pandas/io/common.py

```diff
@@ -747,6 +751,21 @@ def get_handle(
                         f"Only one file per ZIP: {zip_names}"
                     )
 
+        # TAR Encoding
+        elif compression == "tar":
+            tar = tarfile.open(handle, "r:*")
```

Comment:
If pandas supports reading from .tar*, users will probably also expect being able to write to .tar*.

Reply:
Implemented in https://github.com/pandas-dev/pandas/pull/44787/commits/5f22df77accc57cfcd9864052db5b58e5dcae2e3 ✓